packages/flutter_tools/lib/src/project.dart

```diff
@@ -861,8 +861,8 @@ $javaGradleCompatUrl
     }
     for (final XmlElement metaData in document.findAllElements('meta-data')) {
       final String? name = metaData.getAttribute('android:name');
-      // External code checks for this string to indentify flutter android apps. 
-      // See https://critique.corp.google.com/cl/667760684 as an example. 
+      // External code checks for this string to indentify flutter android apps.
+      // See https://critique.corp.google.com/cl/667760684 as an example.
```

Comment:
can you replace this with just "cl/6677..." ? That is drop the full domain name.

Reply:
Done